svthalia/concrexit
# Add webauthn 2fa

Closes #3626 .



### Summary

Enable the possibility to use webauthn for 2fa

### How to test

1. Go to 2fa page
2. Click on the second option instead of the first
3. See if it works. (Note that depending on the browser you can run into problems with http on local host as well as it simply not working at all)
